_includes/alert-two-phase-payments.md

```diff
@@ -0,0 +1,5 @@
+{% include alert.html type="warning" icon="warning" header="Two-phase payments" 
```

Comment:
Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6097051)